v4/source/plugins.yaml

```diff
@@ -265,3 +265,54 @@
             Client := model.NewAPIv4Client("https://your-mattermost-url.com")
 
             manifests, resp := Client.GetWebappPlugins()
+  /plugins/install_from_url:
```

Comment:
Maybe move this up next to the  `/plugins` endpoint?

Reply:
Change added. Thanks for reviewing @hanzei!